libs/etterlatte-token-model/src/main/kotlin/no/nav/etterlatte/token/SaksbehandlerMedRoller.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.etterlatte
+
+import no.nav.etterlatte.config.AzureGroup
+import no.nav.etterlatte.token.Bruker
+
+data class SaksbehandlerMedRoller(val saksbehandler: Bruker) {
```

Comment:
Usikker på om jeg synes dette er riktig da det er kun behandling som for now har roller.